core/src/main/resources/operatorsScore-dataproc-gke-t4.csv

```diff
@@ -265,3 +265,5 @@ MapInPandasExec,1.2
 WindowInPandasExec,1.2
 RoundCeil,3.65
 RoundFloor,3.65
+BloomFilterMightContain,3.16
+BloomFilterAggregate,3.16
```

Comment:
Why is this 3.16 but other operator scores are 3.65?

Reply:
Good catch @cindyyuanjiang ! I fixed it.
That's a default value. So, it won't matter.
But you are right that it should match the change we do in other files to be consistent.